src/lib/definitions.js

```diff
@@ -768,6 +768,9 @@ var createParseIntArgDefault;
         ],
         'D':
         [
+            // The escaped character may be either ] or }
+            define('escape((+("10"+(RP_5_N+FILTER+0)[40]+0+0+0)+FILTER)[40])[2]'),
```

Comment:
I changed it into 1000(d/e/0)0 because it looks shorter and generates the code of same length